flowetl/tests/integration/test_fixuid.py

```diff
@@ -0,0 +1,44 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# -*- coding: utf-8 -*-
+"""
+Tests that fixuid works as expected
+"""
+
+
+def test_uid(docker_client, flowetl_tag):
+    """
+    test that we can run the flowetl container with a specific user
+    """
+
+    user = "1002:1003"
+    out = docker_client.containers.run(
+        f"flowminder/flowetl:{flowetl_tag}", "bash -c 'id -u'", user=user
+    )
+    assert out.decode("utf-8").strip() == "1002"
+
+
+def test_gid(docker_client, flowetl_tag):
+    """
+    test that we can run the flowetl container with a specific user
```

Comment:
```suggestion
    Test that the flowetl container contains a user with a specific UID/GID.
```

Reply:
That's not what's happening really...we are testing that when the container starts the airflow user gets changed to have the specified UID/GID...

Reply:
Hmm, I see what you mean. But isn't that a technical detail of how the presense of this user comes about, and the _purpose_ of the test is to check that this user is present when the container starts up?

Reply:
Well no cos the `airflow` user will always be present - the check is that the `airflow` user has the specific uid/gid?